tests/unit/sftp_test.py

```diff
@@ -63,3 +82,39 @@ def test_get(dst_path, src_path, sftp_session, transmit_payload):
     """Check that SFTP file download works."""
     sftp_session.get(str(src_path), str(dst_path))
     assert dst_path.read_bytes() == transmit_payload
+
+
+def test_get_existing(dst_exists_path, src_path, sftp_session, transmit_payload):
+    """Check that SFTP file download works when target file exists."""
+    sftp_session.get(str(src_path), str(dst_exists_path))
+    assert dst_exists_path.read_bytes() == transmit_payload
+
+
+def test_put_existing(dst_exists_path, src_path, sftp_session, transmit_payload):
+    """Check that SFTP file upload works when target file exists."""
+    sftp_session.put(str(src_path), str(dst_exists_path))
+    assert dst_exists_path.read_bytes() == transmit_payload
+
+
+@pytest.fixture
+def src_path_large(tmp_path, sized_payload):
+    """Return a remote path to a 1025 byte-sized file.
+
+    The pylibssh chunk size is 1024 so the test needs a file that would
+    execute at least two loops.
+    """
+    path = tmp_path / 'large.txt'
+    path.write_bytes(sized_payload)
+    return path
+
+
+def test_put_large(dst_path, src_path_large, sftp_session, sized_payload):
+    """Check that SFTP can upload large file."""
+    sftp_session.put(str(src_path_large), str(dst_path))
+    assert dst_path.read_bytes() == sized_payload
+
+
+def test_get_large(dst_path, src_path_large, sftp_session, sized_payload):
+    """Check that SFTP can download large file."""
+    sftp_session.get(str(src_path_large), str(dst_path))
+    assert dst_path.read_bytes() == sized_payload
```

Comment:
With https://github.com/ansible/pylibssh/pull/638/files#r1927502223, this won't be necessary:
```suggestion
```